get-started/part3.md

```diff
@@ -146,10 +146,36 @@ Now let's run it. You have to give your app a name. Here, it is set to
 docker stack deploy -c docker-compose.yml getstartedlab
 ```
 
-See a list of the 5 containers you just launched:
+Our single service stack is running 5 container instances of our deployed image on one host. Let's investigate.
```

Comment:
Nit: We try to wrap lines around 80 characters. In Atom you can wrap a paragraph by doing Cmd+Ctrl+Q on Mac. Not sure on Linux.